packages/components/src/components/hds/form/super-select/multiple/base.hbs

```diff
@@ -0,0 +1,77 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<div class={{this.classNames}} {{style --hds-form-super-select-dropdown-max-width=this.dropdownMaxWidth}}>
```

Comment:
I was experimenting with the maxWidth a bit and thinking maybe we should set a default value of `calc(100vw - 16px)`? (100% of the screen width minus a bit of side-spacing) @majedelass @heatherlarsen @jorytindall @andgen404 

<img width="798" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/fbe57c9a-7d5f-47f4-b3fb-0172a4f5d19c">

Reply:
I've since refactored `dropdownMaxWidth` to set `matchTriggerWidth` to false automatically if a dropdown maxWidth value is added so this may affect the above idea for setting a default maxWidth...

Reply:
I don't think setting a default maxWidth would work any longer after the refactor mentioned above which automatically ties it to matchTriggerWidth so I'm resolving this thread.